db/seeds.rb

```diff
@@ -34,7 +37,8 @@
     logo: "https://assets.coingecko.com/coins/images/825/large/bnb-icon2_2x.png?1644979850",
     price: 396.63,
     price_change_24h: -9.531828542664,
-    price_change_percentage_24h: -2.34678
+    price_change_percentage_24h: -2.34678,
+    quantity: 3.0
     },
     {
     user_id: 1,
```

Comment:
Be careful with this hard-coded id. If you have issues with your seeds in the future, keep this in mind.